IO/build.gradle

```diff
@@ -12,19 +12,6 @@ dependencies {
     testImplementation libs.jmock.imposters
 }
 
-sourceSets {
-    main {
-        resources {
-            srcDir 'src/main/java' exclude '**/*.java'
```

Comment:
There is only one non-.java file in IO/src, and that is IO.gwt.xml in src/main/resources.